src/TodoListItem.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+function TodoListItem(props) {
+  const { todo } = props;
+  return (
+    <li key={todo.id}>
```

Comment:
There's no harm in using the key here as well, but is not really doing anything.